R/D300.R

```diff
@@ -354,7 +356,9 @@ parse_D300_metadata_file <- function(metadata_file) {
 
 fill_NA <- function(x, from, with) {
   idx <- is.na(x[[from]])
-  x[idx, from] <- x[idx, with]
+  if (any(idx)) {
```

Comment:
To simplify, replace 358L-361L by:
`x[, from := ifelse(is.na(get(from)), get(with), get(from))]`

Reply:
Unfortunately this solution doesn't work:
`Error in 'get(from)': invalid first argument`